tests/utils/__init__.py

```diff
@@ -0,0 +1,73 @@
+from datacube.model import Dataset, DatasetType, MetadataType
```

Comment:
I deleted this file, functions got moved to `util.py`, looks like bad merge

Reply:
I'll check it out...